stable/rabbitmq-ha/templates/role.yaml

```diff
@@ -4,7 +4,7 @@ kind: Role
 metadata:
   labels:
     app: {{ template "rabbitmq-ha.name" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```

Comment:
This would be better to change all templates to use a function from `_helpers.tpl`. Can you copy the `"rabbitmq-ha.chart"` function generated by `helm create` and use that everywhere?

Reply:
Done. I've used `template` instead of `include` to call the function to stay consistent with rest of the calls.